src/core/public/public.api.md

```diff
@@ -313,12 +313,12 @@ export interface OverlayStart {
     // Warning: (ae-forgotten-export) The symbol "React" needs to be exported by the entry point index.d.ts
     // 
     // (undocumented)
-    openFlyout: (flyoutChildren: React.ReactNode, flyoutProps?: {
+    openFlyout: (flyoutChildren: React_2.ReactNode, flyoutProps?: {

```

Comment:
Hmm, these don't seem like an improvement

Reply:
Looks like this is happening because **src/core/public/overlays/overlay_service.ts** refers to the `React` symbol without an `import` statement.  Thus the symbol refers to the global window variable (provided by React for legacy apps) instead of a regular import.  API Extractor renames the symbol because it cannot be sure that refers to the "same" React as a regular import.  (For example, if you were loading side-by-side versions of React 15 and 16, then the version mapping is indeterminate without knowing your linker configuration.  Whereas for an import, the module resolution rules give a well-defined answer.)

There does seem to be a bug with the renaming, though.  API Extractor should have renamed the imported symbol, not the global variable.  Feel free to open a GitHub issue if you like.  :-)

Reply:
Thanks for the feedback! We don't need to use the global, so this should fix our issues, but I've created https://github.com/microsoft/web-build-tools/issues/1316